scrapper.py

```diff
@@ -1,10 +1,143 @@
 import requests
 from bs4 import BeautifulSoup
+from sys import argv
+import re
 
-page = requests.get('http://127.0.0.1:5500/krisha_scrapper/flat_list.html')
-soup = BeautifulSoup(page.content, 'html.parser')
+def check_cli_arg(command_line_arguments: list) -> str:
+    """ Function checks:
+            - that URL was entered on command line, if not promts entering URL
```

Comment:
- that URL was entered in a command line. If it wasn't than a user gets a respective message and is prompted to provide URL as an argument.